lib/reporters/checkstyle.js

```diff
@@ -0,0 +1,76 @@
+/*
+ * Author: Josh Hagins
+ * https://github.com/jawshooah
+ *
+ * Modified from the original by: Boy Baukema
+ * https://github.com/relaxnow
+ */
+
+module.exports = {
+  reporter: function(results) {
```

Comment:
Is there a checkstyle module that could be reused, instead of implementing it "from scratch"?